src/playitem.js

```diff
@@ -24,6 +26,15 @@ var playItem = function(player, autoadvanceTimeout, obj) {
     player.addRemoteTextTrack(obj.textTracks[i]);
   }
 
+  if (obj.cuePoints && obj.cuePoints.length) {
+    track = player.addRemoteTextTrack({kind: 'metadata'});
```

Comment:
`addRemoteTextTrack` returns a `HTMLTrackElement`, so, you want to set this variable to `trackEl`;
Then `track` should be set to `trackEl.track`.